packages/nimble-components/src/anchor-tabs/index.ts

```diff
@@ -70,14 +76,41 @@ export class AnchorTabs extends FoundationElement {
      */
     public tablist!: HTMLElement;
 
+    /**
+     * @internal
+     */
+    public readonly leftScrollButton!: Element;
+
+    /**
+     * @internal
+     */
+    public readonly tabSlotName = 'anchortab';
+
+    private readonly tabListResizeObserver: ResizeObserver;
     private tabIds: string[] = [];
 
+    public constructor() {
+        super();
+        this.tabListResizeObserver = new ResizeObserver(entries => {
+            let tabListVisibleWidth = entries[0]?.contentRect.width;
+            if (tabListVisibleWidth !== undefined) {
+                const buttonWidth = this.leftScrollButton?.clientWidth ?? 0;
+                tabListVisibleWidth = Math.ceil(tabListVisibleWidth);
+                if (this.showScrollButtons) {
+                    tabListVisibleWidth += buttonWidth * 2;
+                }
+                this.showScrollButtons = tabListVisibleWidth < this.tablist.scrollWidth;
+            }
+        });
+    }
+
     /**
      * @internal
      */
     public activeidChanged(_oldValue: string, _newValue: string): void {
```

Comment:
For the anchor tabs, would it make more sense to scroll a tab into view when it is navigated to rather than waiting for the activeid to change? As is, you can't always see what tab you're going to activate when you press Enter.

Reply:
So, navigating will already bring the tab into view (via arrow keys). All this implementation does is bring the tab into view via a programmatic change to `activeid`.

Reply:
For the anchor tabs too? That wasn't the behavior I saw in Edge.